keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1083,10 +1117,15 @@ Below are some examples to consider, in addition to the aforementioned [maturity
 - Re-evaluate introduction of a generic opinionated condition type
   indicating that a pod should be retried (see: [Evolving condition types](#evolving-condition-types))
 - Simplify the code in job controller responsible for detection of failed pods
-  based on the fix for pods stuck in the running phase (see: [Marking pods as Failed](marking-pods-as-failed))
+  based on the fix for pods stuck in the running phase (see: [Marking pods as Failed](marking-pods-as-failed)).
+  Also, introduce a worker in the disruption controller to mark pods
+  stuck in the pending phase, with set deletionTimestamp, as failed.
 - Commonize the code for appending pod conditions between components
 - Do not update the pod disruption condition (with type=`DisruptionTarget`) if
   it is already present with `status=True`
+- Re-evaluate the decision about handling failed appending of a pod disruption
```

Comment:
I don't think this should be under discussion. Interrupting the delete operation sounds best.

Reply:
Indeed, deleted the point.